### PR TITLE
Creats new tests directory and test file for a generic API call

### LINKED ISSUE #14: Build generic API test for practice with Django

### PR DESCRIPTION
**Pull request by TravisGM92**
### Participating Group Members:
* Travis McKinstry 
 
### Code Highlights:
   * Creates first test for a generic API call
 
### Have Tests Been Added?
- [ ] No.
- [ ] Yes, but all tests are not passing. 
- [x] Yes, and all are passing.
 
### Message/Questions for reviewer:

To run tests, navigate to `main_app/` in terminal and run the following commands: `python3 ../manage.py test tests`
 
### Issues: 
* Addresses #
* Closes #
 
### Screenshots (if appropriate): 
*
 
### Tracking Consistency:
- [x] added appropriate labels
- [x] My code follows the code style of this project and has removed all unnecessary annotations
- [x] I have added comments on my pull request, particularly in hard-to-understand areas
- [x] All new and existing tests passed

- [x] looked at PR preview to check spelling, syntax, formatting, and completion

### CODE REVIEW

File: main_app/tests/test_app.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+import requests
+
+
+class AnimalTestCase(TestCase):
+
+    def test_can_make_api_call(self):
+        """Can make an API call and expect specific attributes"""
+        data = requests.get('https://jsonplaceholder.typicode.com/todos/1').json()
+        # import code; code.interact(local=dict(globals(), **locals()))
+        self.assertEqual(str(type(data)), "<class 'dict'>")
```
Comment (judithpillado) [inline main_app/tests/test_app.py:11]: What does class 'dict' stand for or what does it mean?

**Review by judithpillado — APPROVED** [review]

File: main_app/tests/test_app.py (hunk 1 above)
Comment (TravisGM92) [reply main_app/tests/test_app.py:11]: @judithpillado It's what Python calls a "hash". Technically in Python it's a 'dictionary', but abbreviated here as 'dict'. Ruby calls it a "hash"

Comment (judithpillado) [reply main_app/tests/test_app.py:11]: oh okay thanks for explaining! Awesome work

**Comment by robertorodriguez12** [comment]
Good work!

**Pull request merged**